bitcoin/src/blockdata/constants.rs

```diff
@@ -84,7 +84,7 @@ fn bitcoin_genesis_tx() -> Transaction {
 
     // Inputs
     let in_script = script::Builder::new()
-        .push_int(486604799)
+        .push_uint(486604799)
```

Comment:
In 1100de47bb893407bd2380433460085d4388c282:

We should just use `unwrap` here and make `push_uint` `pub(super)`. In general I am suspicious of `pub(in module)` because it suggests there is some abstraction failure. In this case it's just to save an unwrap call which is guaranteed to be optimized away.

Alternately we can use `push_int_unchecked`